readme.md

```diff
@@ -248,10 +252,8 @@ yarn lint
 yarn unit
 yarn smoke
 
-## run closure compiler (on whitelisted files)
-yarn closure
-## import your report renderer into devtools-frontend and run devtools closure compiler
-yarn compile-devtools
+## run tsc compiler
+yarn type-check
```

Comment:
mind also updating the mention of "closure annotations" in CONTRIBUTING.md too? Can link to https://github.com/Microsoft/TypeScript/wiki/JSDoc-support-in-JavaScript instead of that closure link

Reply:
done